codemeta.json

```diff
@@ -0,0 +1,597 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "author": [
+        {
+            "@id": "https://orcid.org/0000-0003-4568-7005",
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St., 02138 Cambridge MA, US"
+            },
+            "familyName": "Donath",
+            "givenName": "Axel"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "HeidelbergCement AG, Berliner Stra\u00dfe 6, 69120 Heidelberg, Germany"
+            },
+            "familyName": "Deil",
+            "givenName": "Christoph"
+        },
+        {
+            "@id": "https://orcid.org/0000-0002-8219-4667",
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Astroparticle and Cosmology Laboratory CNRS, Universit\u00e9 de Paris, 10 Rue Alice Domon et L\u00e9onie Duquet, 75013 Paris, France"
```

Comment:
the name of his affiliation is:
"Astroparticle and Cosmology Laboratory CNRS, Universit\u00e9 de Paris Cit\u00e9, 10 Rue Alice Domon et L\u00e9onie Duquet, 75013 Paris, France"